packages/token-search-discovery-controller/CHANGELOG.md

```diff
@@ -7,22 +7,24 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.0.0]
+
 ### Added
 
-- Introduce the `logoUrl` property to the `TokenSearchApiService` response
+- Introduce the `logoUrl` property to the `TokenSearchApiService` response ([#5195](https://github.com/MetaMask/core/pull/5195))
   - Specifically in the `TokenSearchResponseItem` type
-- Introduce `TokenDiscoveryApiService` to keep discovery and search responsibilities separate
+- Introduce `TokenDiscoveryApiService` to keep discovery and search responsibilities separate ([#5214](https://github.com/MetaMask/core/pull/5214))
   - This service is responsible for fetching discover related data
   - Add `getTrendingTokens` method to fetch trending tokens by chain
   - Add `TokenTrendingResponseItem` type for trending token responses
 - Export `TokenSearchResponseItem` type from the package index
 
 ### Changed
 
-- Update the TokenSearchApiService to use the updated URL for `searchTokens`
+- Bump @metamask/utils to v11.1.0 ([#5223](https://github.com/MetaMask/core/pull/5223))
+- Update the `TokenSearchApiService` to use the updated URL for `searchTokens` ([#5195](https://github.com/MetaMask/core/pull/5195))
+- **BREAKING:** The `searchTokens` method now takes a `query` parameter instead of `name`
```

Comment:
Does this need a PR link too?

``` suggestion
- **BREAKING:** The `searchTokens` method now takes a `query` parameter instead of `name` ([#5195](https://github.com/MetaMask/core/pull/5195))
```